config/webpack.dev.js

```diff
@@ -35,6 +35,7 @@ module.exports = (env) => {
                 'base-scss': path.resolve(appBase, 'src/stylesheets/'),
                 'rs-scss': path.resolve(appBase, 'src/vendor/react-store/stylesheets/'),
             },
+            symlinks: false,
```

Comment:
is this value not required in webpack.prod.js?

Reply:
No